R/common-find-variation.R

```diff
@@ -425,4 +425,103 @@ getPipedValue <- function(string) {
   } else {
     result
   }
+}
+
+
+#' @title
+#' Find variation across groups
+#' @description Plot a boxplot to compare the variation across the groups. 
```

Comment:
From testing, it plots it in red (versus black if not statistically different).  Maybe this should be included in the documentation.